compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/NodeKind.java

```diff
@@ -58,6 +58,7 @@
     TABLE_MULTI_KEY,
     RETRY_SPEC,
     CLASS_DEFN,
+    TESTABLE_PACKAGE,
```

Comment:
As discussed offline, I don't think we need this. Even if we do need to differentiate testable packages, there's already a flag `org.ballerinalang.model.elements.Flag#TESTABLE`.